docs/execplans/1-4-6-model-routed-transactions-and-session-gating.md

```
@@ -272,42 +274,54 @@ All commands run from the repository root.
 
 Edit `crates/mxd-verification/Cargo.toml`:
 
-    [dependencies]
-    stateright = "0.30"
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)